src/Symfony/Component/Dotenv/Command/DebugCommand.php

```diff
@@ -47,6 +48,22 @@ public function __construct(string $kernelEnvironment, string $projectDirectory)
         parent::__construct();
     }
 
+    protected function configure(): void
+    {
+        $this
+            ->setDefinition([
+                new InputArgument('name', InputArgument::OPTIONAL, 'The name of variable to filter for debug.', null, $this->getAvailableVars()),
```

Comment:
For lazy-loading, you can use the closure. 
```suggestion
                new InputArgument('name', InputArgument::OPTIONAL, 'The name of variable to filter for debug.', null, $this->getAvailableVars(...)),
```

Reply:
ah, true, fixed!

Reply:
Thanks